cmd/tk/jsonnet.go

```diff
@@ -27,7 +27,7 @@ func evalCmd() *cli.Command {
 		jsonnetOpts := tanka.Opts{
 			JsonnetOpts: getJsonnetOpts(),
 		}
-		jsonnetOpts.EvalPattern = *evalPattern
+		jsonnetOpts.EvalScript = strings.Join([]string{"(import '%s')", *evalPattern}, ".")
```

Comment:
```suggestion
		jsonnetOpts.EvalScript = "(import '%s')."+*evalPattern
```

No need for slices and `strings.Join`